nextflow_schema.json

```diff
@@ -314,6 +314,13 @@
                     "hidden": true,
                     "fa_icon": "fas fa-align-justify"
                 },
+                "maximum_process_num": {
+                    "type": "integer",
+                    "default": 1000,
+                    "fa_icon": "fas fa-cogs",
+                    "description": "Maximum number of processes for each allele generated by the protein prediction. (If 2 alleles are chosen and maximum_process_num is 1000 the pipeline is allowed to start 2000 processes.)",
```

Comment:
Could you not let the user define the total number of tasks that should be created and then divide it by the number of alleles?

Reply:
![](https://media1.tenor.com/m/3xoRK7hFE3gAAAAC/patrick-star-spongebob-squarepants.gif)

Thats a really good Idea! Will do and update the PR also with the name change of the parameter!